releasenotes/notes/aer-provider-config-backend-3a89b7587c34c6ee.yaml

```diff
@@ -0,0 +1,43 @@
+---
+upgrade:
```

Comment:
This note reads like a feature note instead of an upgrade note. 

That being said I actually worry about the backwards compat for this. Previously the kwargs would be used as filters that get passed to `backends`. This is used to set filters on `backend.configuration()` which is useful for real backends with immutable configurations, for aer I'm not sure how this would be used. But if you want to make the breaking change you need to call it out as such